lib/react_client/react_interop.dart

```diff
@@ -149,6 +159,7 @@ class ReactElement {
 class ReactComponent {
   external Component get dartComponent;
   external InteropProps get props;
+  external InteropProps get state;
```

Comment:
Is this a typo, or are `props` and `state` both instances of `InteropProps`?

Reply:
Ooh good catch, this should be JsMap and maybe InteropProps should extend JsMap so we can remove some of those casts. I'll look into that

Reply:
Ooh good catch, this should be JsMap and maybe InteropProps should extend JsMap so we can remove some of those casts. I'll look into that